### ISSUE TITLE
SNMP: add alias tag

### ISSUE THREAD

**Issue by r0bj** [influxdata/telegraf#911]
There is "ifName" OID (1.3.6.1.2.1.31.1.1.1.1) but also "ifAlias" (1.3.6.1.2.1.31.1.1.1.18) as a custom port description. It would be great to have ability add additional tag as alias.

Maybe as an additional `mapping_table` in config?

ifName:
```
snmpwalk -v2c -c public 127.0.0.1 1.3.6.1.2.1.31.1.1.1.1.8
iso.3.6.1.2.1.31.1.1.1.1.8 = STRING: "eth2"
```
ifAlias:
```
snmpwalk -v2c -c public 127.0.0.1 1.3.6.1.2.1.31.1.1.1.18.8
iso.3.6.1.2.1.31.1.1.1.18.8 = STRING: "GTS"
```

Example of line protocol:
```
ifHCInOctets,host=gw1,instance=eth0,alias=GTS,unit=octets ifHCInOctets=378553758147i 1458676458140933859
```

**Comment by titilambert** [comment]
This is a goof idea ! In fact, we have to add the string data support in snmp plugin... Also, All string data could be processed as influxdb tag or as influxdb value.

@sparrc Is there any issue to send string data as value in influxdb from Telegraf

**Comment by titilambert** [comment]
To get this working we need to get the issue #808 done first ( be able to merge dara in the same measurement )

**Comment by sparrc** [comment]
Nope, strings are allowed in the database as both fields and tags

**Comment by schotlandzegtja** [comment]
Hey, I wonder if there has been any movement on this?

This would be really handy to have. Not sue where I would need to start if I were to submit a patch myself, if someone could make any suggestions?

**Comment by sparrc** [comment]
with the new SNMP plugin (https://github.com/influxdata/telegraf/pull/1389) the user can now specify SNMP OIDs to be used as tags, which I believe fixes this issue.

It will be available in release 1.0
